python/pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[build-system]
+requires = ["setuptools>=61.0", "setuptools_scm[toml]>=6.2", "pep440"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pyvsag"
+description = "vsag is a vector indexing library used for similarity search"
+authors = [
+    {name = "the vsag project", email = "the.vsag.project@gmail.com"},
+]
+license = {text = "Apache-2.0"}
+keywords = ["search", "nearest", "neighbors"]
+classifiers = [
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
```

Comment:
@jac0626, can this refactoring also support packaging pyvsag for python 3.6 and 3.7